### PR TITLE
Update monitor-built-in-alerting.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by just1900 — APPROVED** [review]
LGTM

File: tidb-cloud/monitor-built-in-alerting.md [hunk 1]
```diff
@@ -5,18 +5,54 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
 
 ## Subscribe to alert notification emails
 
 If you are a member of a project and you want to get alert notification emails of clusters in your project, take the following steps:
 
+1. In the **Alerts** page , click the "Subscribe Alerts" button. 
+2. Enter your email address, and then click **Subscribe**. 
+    > **Tip:**
+    >
+    > The alert subscription is for all the alerts in current project. If you have multiple clusters in the project, you just need to subscribe once.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:52]:
```suggestion
    > The alert subscription is for all alerts in the current project. If you have multiple clusters in the project, you just need to subscribe once. 
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 2]
```diff
@@ -5,18 +5,54 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
+2. Click Switch button to disable or enable alert rules.
+3. Click the "Edit" button to update the threshold of alert rules. 
+
+    > **Tip:**
+    >
+    > TiDB Cloud provide limited capability for alert rule editing . Some alert rules does not support to edit. If you would like to configure different trigger conditions, thresholds, or frequency, or have alerts automatically trigger actions in downstream services like [PagerDuty](https://www.pagerduty.com/docs/guides/datadog-integration-guide/), consider using a third-party monitoring and alerting integration. Currently, TiDB Cloud supports the [Datadog integration](/tidb-cloud/monitor-datadog-integration.md) and the [Prometheus and Grafana integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
 
 ## Subscribe to alert notification emails
 
 If you are a member of a project and you want to get alert notification emails of clusters in your project, take the following steps:
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:46]:
```suggestion
To get alert notification emails of clusters in your project, take the following steps:
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 3]
```diff
@@ -5,18 +5,54 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
 
+1. In the **Alerts** page , click the "Subscribe Alerts" button. 
+2. Enter your email address, and then click **Subscribe**. 
+    > **Tip:**
+    >
+    > The alert subscription is for all the alerts in current project. If you have multiple clusters in the project, you just need to subscribe once. 
+
+You can also add the subscription from the Alert Subscription page, take the following steps:
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:54]:
```suggestion
Alternatively, you can also add the subscription from the **Alert Subscription** page as follows:
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 4]
```diff
@@ -5,18 +5,54 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
+TiDB Cloud allows you to edit the rules for the Build-in Alerts. You can disable/enable the alerts or update the threshold. 
+
+To edit the alert rules on the Alerts page, take the following steps:
+1. In the **Alerts** page , click the "Edit Rules" button. 
+2. Click Switch button to disable or enable alert rules.
+3. Click the "Edit" button to update the threshold of alert rules. 
+
+    > **Tip:**
+    >
+    > TiDB Cloud provide limited capability for alert rule editing . Some alert rules does not support to edit. If you would like to configure different trigger conditions, thresholds, or frequency, or have alerts automatically trigger actions in downstream services like [PagerDuty](https://www.pagerduty.com/docs/guides/datadog-integration-guide/), consider using a third-party monitoring and alerting integration. Currently, TiDB Cloud supports the [Datadog integration](/tidb-cloud/monitor-datadog-integration.md) and the [Prometheus and Grafana integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
```
[the remark this answers is not in the document]
Comment (qqqdan) [reply tidb-cloud/monitor-built-in-alerting.md:37]: 是的

File: tidb-cloud/monitor-built-in-alerting.md [hunk 5]
```diff
@@ -5,17 +5,48 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
+    > If you have multiple projects, you can switch to the target project from the ☰ hover menu in the upper-left corner.
+
+2. Click the name of the target cluster. The cluster overview page is displayed.
+3. Click **Alerts** in the left navigation pane.
+4. The **Alerts** page displays the active alerts by default. You can view the information of each active alert such as the alert name, trigger time, and duration. 
+5. If you also want to view the closed alerts, just click the **Status** drop-down list and select **Closed** or **All**.
+
+## Edit alert rules
+
+In TiDB Cloud, you can edit the alert rules by disabling or enabling the alerts or updating the alert threshold.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:29]:
```suggestion
In TiDB Cloud, you can edit the alert rules by disabling or enabling the alerts or updating the alert threshold.

```

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/monitor-built-in-alerting.md [hunk 6]
```diff
@@ -50,16 +83,14 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Total TiKV node CPU utilization exceeded 80% for 10 minutes | Total TiKV node CPU utilization of cluster ABC in project XYZ has exceeded 80% for 10 minutes. Consider increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
+| Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Total TiFlash node CPU utilization of cluster ABC in project XYZ has exceeded 80% for 10 minutes. Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |
+| TiKV storage utilization exceeds 80% | Total TiKV storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiKV to increase your storage capacity. |
+| TiFlash storage utilization exceeds 80% | Total TiFlash storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:93]: 前半句话和 condition 基本一样，有点冗余，建议删掉
```suggestion
| Total TiDB node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiDB to reduce the memory usage percentage of the current workload.|
| Total TiKV node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
| Total TiFlash node memory utilization across cluster exceeded 70% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the memory usage percentage of the current workload. |
| Total TiDB node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload.|
| Total TiKV node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
| Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |
| TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. |
| TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. |
```

Comment (qiancai) [reply tidb-cloud/monitor-built-in-alerting.md:93]: @qqqdan PTAL

File: tidb-cloud/monitor-built-in-alerting.md [hunk 7]
```diff
@@ -5,17 +5,53 @@ summary: Learn how to monitor your TiDB cluster by getting alert notification em
...
-The TiDB Cloud built-in alerting feature provides you with an easy way to be notified by emails whenever a TiDB Cloud cluster in your project triggers one of TiDB Cloud built-in alert conditions.
+TiDB Cloud provides you with an easy way to view alerts, edit alert rules, and subscribe to alert notification emails.
 
-This document describes how to subscribe to alert notification emails from TiDB Cloud and also provides the TiDB Cloud built-in alert conditions for your reference.
+This document describes how to do these operations and provides the TiDB Cloud built-in alert conditions for your reference.
 
-## Limitation
+> **Note:**
+>
+> Currently, the Alerts page is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:14]:
```suggestion
> Currently, the alert feature is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

File: tidb-cloud/serverless-tier-limitations.md
Comment (ran-huang) [inline tidb-cloud/serverless-tier-limitations.md:1]: updated this doc.

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Pull request merged**